### DISCUSSION TITLE
RTX 7900XT

### DISCUSSION THREAD

**Discussion by rumpelstilz69** [SpiralLinux/SpiralLinux-project#430] [Q&A]
Hi, my Spirallinux doesn't boot anymore after the upgrade to an RX 7900 XT, also with the latest 12.5 Live ISO! I was thinking that's no problem because the Kernel you use. Can i do something, or must i change the Distro!?
Greetings Martin

**Comment by g4bb3r** [comment]
Doesn't boot or boot without image?
If boot without image it's the driver fault, you need this https://www.linuxcapable.com/install-nvidia-drivers-on-debian/

**Reply by rumpelstilz69** [reply]
Thx, but i have a AMD Graphics Card! 
Lg Martin

**Reply by g4bb3r** [reply]
My bad...
Install mesa, llvm, amdgpu, linux-firmware. and
sudo rm /etc/modprobe.d/blacklist-amdgpu.conf

If this doesn't work, try add 
Section "Device"
      Identifier "modesetting"
      Driver "modesetting"
EndSection
to /etc/X11/xorg.conf.d/99-modesetting.conf

Still not working?
https://www.amd.com/en/support/linux-drivers

**Reply by rumpelstilz69** [reply]
Thx, i will try it, i'm on Fedora for now, the Problem is, that de usb Stick doesn't boot, maybe i must change the Grub settings to start the Live USB Stick.

**Reply by AlphaElwedritsch** [reply]
It's a BIOS setting to boot from USB stick
It's before grub is booting

**Reply by rumpelstilz69** [reply]
I know how to boot en USB Stick, but Spiral doesn't boot correctly, that's why i was thinking, maybe i can try to make en Grub Boot Option, when i boot Spiral!
But anyway, i found today a very intresting Distro, Ublue OS (Universal Blue)! So first i try out the bazzite Version and maybe come later back to Bookworm!

**Comment by AlphaElwedritsch** [comment]
> but Spiral doesn't boot correctly


Maybe a matter of that:

> ⚠️ Important: Rufus, LinuxLive USB Creator (LiLi), Ventoy, and Unetbootin are NOT recommended by SpiralLinux for creating the bootable USB device due to many reported issues.

**Reply by rumpelstilz69** [reply]
Thx, i read this, but i haven't no issues with Ventoy so far, i idon't know i tryout out de coventionell with balena Etcher,. dd on so on! I will try it!
